specifyweb/frontend/js_src/lib/components/QueryBuilder/Header.tsx

```diff
@@ -80,7 +80,10 @@ export function QueryHeader({
   const [isBasic, setIsBasic] = useQueryViewPref(query.id);
 
   return (
-    <header className="flex flex-col items-center justify-between gap-2 overflow-x-auto whitespace-nowrap sm:flex-row sm:overflow-x-visible">
+    <header
+      className="flex flex-col items-center justify-between gap-2
```

Comment:
fyi, using `"` for multi-line strings is discouraged as it's not well supported by all tools. If you need multi-line, use **{\`** and **\`}** instead (also, please indent everything correctly - that helps with readability and consistency - unfortunately prettier and eslint don't do that automatically in this caes